cilium/cilium.io
# Add in production section

This pull request introduces a small update by changing the grid cards to a slider on the home page.

Steps to check:

- open [preview](https://deploy-preview-533--cilium.netlify.app/)
- make sure that everything looks and feels as expected
